pkg/testutil/scenarios.go

```diff
@@ -15,22 +15,18 @@ import (
 	"time"
 )
 
-// TestScenario represents a standard test case to be validated.
+// CLIScenario represents a CLI test case to be validated.
 //
 // Most of the fields in this struct are optional; if they are not
-// provided RunScenario will not apply the behavior indicated for
+// provided RunCLIScenario will not apply the behavior indicated for
 // those fields.
-type TestScenario struct {
+type CLIScenario struct {
 	// API is a mock API implementation which can be used by the
 	// command under test
 	API mock.API
-	// Arg is the input arguments for the command to execute (not
+	// Args is the input arguments for the command to execute (not
 	// including the command names themselves).
-	Arg string
-	// Args is the input arguments for the command to execute.
-	// DEPRECATED: Args will be removed once all users are using
-	// RunScenario() or RunScenarios().
-	Args []string
+	Args string
```

Comment:
Part 2 🚗: This might have to change back to `[]string` to agree with `os.Cmd`, and that probably means using `SplitArgs()` in more places if you were to use my feedback.